test/Y2ModuleConfig_test.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+require "yaml"
+
+Yast.import "Y2ModuleConfig"
+Yast.import "Desktop"
+Yast.import "Profile"
+
+include Yast::Logger
+
+FIXTURES_PATH = File.join(File.dirname(__FILE__), 'fixtures')
+DESKTOP_DATA = YAML::load_file(File.join(FIXTURES_PATH, 'desktop_files', 'desktops.yml'))
+
+describe Yast::Y2ModuleConfig do
+  describe "#unhandled_profile_sections" do
+    let(:profile_unhandled) { File.join(FIXTURES_PATH, 'profiles', 'unhandled_and_obsolete.xml') }
+
+    it "returns all unsupported and unknown profile sections" do
+      Yast::Profile.ReadXML(profile_unhandled)
+      Yast::Y2ModuleConfig.instance_variable_set("@ModuleMap", DESKTOP_DATA)
```

Comment:
I actually wanted to use allow(...) but that doesn't seem to work for variables
I'd like to refactor it later to be a lazy-loading method instead of a variable as this variable is filled in constructor
My solution is closer to allow(...), that's why I used that and I'd rather keep it